app/controllers/api/motions_controller.rb

```diff
@@ -11,6 +11,22 @@ def close
     respond_with_resource
   end
 
+  def create_outcome
+    load_and_authorize(:motion, :create_outcome)
+    MotionService.create_outcome(motion: @motion,
+                                 params: permitted_params.motion,
+                                 actor:  current_user)
+    respond_with_resource
+  end
+
+  def update_outcome
+    load_and_authorize(:motion, :update_outcome)
```

Comment:
How about making the service smart enough to accept an id, so the controller doesn't have to know about a Motion at all?